src/Write/Event/FeatureWasCreated.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Core\Toggle\Write\Event;
+
+final class FeatureWasCreated
+{
+
+    private string $eventType;
+
+    private string $featureId;
```

Comment:
We can define properties in the same order as they're assigned in the constructor:
```suggestion
    private string $featureId;
    private string $eventType;
```